VideoLocker/res/values/strings.xml

```diff
@@ -483,6 +483,8 @@
     <!-- Registration Screen -->
     <!-- Create account agreement. Followed by list of agreement names like "End User License Agreement" -->
     <string name="by_creating_account">By creating an {platform_name} account, you agree to the </string>
+    <!-- Create account agreement. Text associated with the edX TOS and honor code link -->
```

Comment:
Something strange is going on with the GitHub diff tool. I addressed your comment, but this is still appearing on this page (I even tried to squash and I am still seeing this here). But if you look at the "files changed" page you can see the new code there.